packages/ff/ff.0.2.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+authors: [ "Danny Willems <be.danny.willems@gmail.com>" ]
+maintainer: [ "Danny Willems <be.danny.willems@gmail.com>" ]
+synopsis: "OCaml implementation of Finite Field operations"
+description: "OCaml implementation of Finite Field operations"
+homepage: "https://gitlab.com/dannywillems/ocaml-ff"
+dev-repo: "git+https://gitlab.com/dannywillems/ocaml-ff.git"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-ff/issues"
+license: "MIT"
+depends: [
+  "dune" {build & >= "2.0"}
+  # Has not been tested with other Zarith versions.
+  # The CI should verify it and it can then be extent
+  "zarith" {build & = "1.9.1"}
```

Comment:
This dependency is not a build dependency. In your own `src/dune` file:
```
(library
 (public_name ff)
 (libraries zarith)
 (name ff))
```
it is used at runtime.

Reply:
Also the dependency is too strict. This should be at least `>= "1.9.1"` if you really want to be prudent, and maybe add `& < "2.0"` if you want to be extra careful if you think the next major version might break your code but otherwise the upper-bound is not necessarily needed.
A lower-bound constraint is nice to have though, otherwise if the dependency is too strict people won't be able to install your package alongside other packages that might require at least a different version.

Reply:
I agree with the upper-bound version. I prefer to have a lower-bound (and leave at 1.9.1 for the moment). I'll update the dep if someone does require 1.9.0 or 1.x. I prefer to be strict first and increase the version interval when required.